packages/grafana-runtime/src/measurement/fps.ts

```diff
@@ -0,0 +1,12 @@
+export const perf = {
+  fps: 0,
+  ok: true,
+};
+
+let lastUpdate = Date.now();
+function measure() {
+  let now = Date.now();
+  perf.fps = 1e3 / (now - lastUpdate);
+  lastUpdate = now;
+}
+setInterval(measure, 50); // 20hz
```

Comment:
i played with a few variants and this value should probably be some multiple of the framerate we want to reduce noise. it's not unusual for it to miss the deadline a bit on 1 cycle and catch up on the next. so here polling every 100ms and throttling when it exceeds 105ms works better for a 50ms target.